en/messages.json

```diff
@@ -252,6 +252,10 @@
       }
     }
   },
+  "openInNoContainer": {
+    "message": "Open in No Container",
```

Comment:
This feels a bit weird. Would it work with `Open in tab without container`, or `Open in standard tab`? Also, I realized recently that buttons should use sentence case, according to Firefox copy guidelines. I wouldn't go and fix them all, but maybe we can fix these two?

Reply:
I used the same wording of the context menu when we right-click on *new tab* button in the tabbar. I agree it sounds weird. 

Looking at the wording in other parts of the UI (context menu when right clicking a tab), we could change both buttons to have something like:

[Open in New Tab] [Open in New [Container Name] Tab]

What do you think of this solution?

Reply:
> I used the same wording of the context menu when we right-click on _new tab_ button in the tabbar. I agree it sounds weird.

Right, but that's stand-alone on top of a list of containers, I don't think it's relevant for this button.

> [Open in New Tab] [Open in New containerName Container Tab]

This would work for me (again, sentence case though). In the end, even if "Open in new tab" might not be 100% clear, it's still a significant improvement on the current behavior.

Reply:
On second thought though: it opens in the same tab, so it might be misleading? "Open without container" and "Open in $containerName$ container" might be an option.

Reply:
Actually, it does open a new tab for the containerized one that replace the regular one. Granted, it isn't visible for the user.

What's bugging me is the negative "without", "no", etc. I guess we could simply use:

[Open in current tab] OR [Open in Work container tab]

Reply:
> [Open in current tab] OR [Open in Work container tab]

Works for me 👍🏼